### DISCUSSION TITLE
心心念念的dark mode啊。。。

### DISCUSSION THREAD

**Discussion by pptech-dev** [ant-design/ant-design-mobile#4624] [Q&A] [answered]
心心念念的dark mode啊。。。rc.8都还没看到

**Comment by awmleer** [comment]
正式版之前应该都不会做的了，目前的时间节奏大概是：1月初开始做 design token 相关的整理和改造，做完之后才有可能渐渐加入 dark mode，所以这么看的话怎么也得到过年后了

**Reply by pptech-dev** [reply]
T_T

**Comment by GOWxx** [comment, marked as answer]
link #5390

**Reply by awmleer** [reply]
+1 @pptech-dev 目前已经基本支持了，可以用起来了

一小部分组件的样式还在微调，但是不影响使用
